src/QueuedWebhook.php

```diff
@@ -628,4 +631,14 @@ public static function cronQueuedWebhookClean(?CronTask $task = null)
         $task->setVolume($vol);
         return ($vol > 0 ? 1 : 0);
     }
+
+    public function post_getFromDB()
+    {
+        parent::post_getFromDB();
+
+        if (!GLPI_WEBHOOK_ALLOW_RESPONSE_SAVING) {
+            // Block viewing response body if saving is disabled by config
+            unset($this->fields['response_body']);
+        }
```

Comment:
I would say that a search option should never be added because this isn't a feature I think that should be encouraged to be used except in very specific debugging cases.